server/lib/generate-alerts/generate-alerts-script.js

```diff
@@ -310,6 +312,51 @@ function generateAlert(params) {
     alert.GeoLocation = randomArrayItem(GeoLocation);
   }
 
+  if (params.office) {
+    const beforeDate = new Date(new Date(alert.timestamp) - 3 * 24 * 60 * 60 * 1000);
+    const IntraID = randomArrayItem(Office.arrayUuidOffice);
+    const InterID = randomArrayItem(Office.arrayUuidOffice);
+    const OrgID = randomArrayItem(Office.arrayUuidOffice);
+    const objID = randomArrayItem(Office.arrayUuidOffice);
+    const userID = randomArrayItem(Office.arrayUuidOffice);
+    const appID = randomArrayItem(Office.arrayUuidOffice);
+
+    alert.rule = randomArrayItem(Office.arrayRulesOffice);
+    alert.decoder = randomArrayItem(Office.arrayDecoderOffice);
+    alert.GeoLocation = randomArrayItem(GeoLocation);
+    alert.data.integration = "Office365";
+    alert.location = Office.arrayLocationOffice;
+    alert.data.office365 = {
+      CreationTime: formatDate(beforeDate, 'Y-M-DTh:m:s.lZ'),
+      Id: IntraID,
+      Operation: "UserLoggedIn",
+      OrganizationId: OrgID,
+      RecordType: "15",
+      ResultStatus: "Success",
+      UserKey: userID,
+      UserType: "0",
+      Version: "1",
+      Workload: "AzureActiveDirectory",
+      ClientIP: "77.231.182.17",
```

Comment:
Would be a good idea to randomize this as well since there are plans to check for location based on this value.